pom.xml

```diff
@@ -9,18 +9,33 @@
 	</parent>
 	<groupId>life.qbic</groupId>
 	<artifactId>projectbrowser-portlet</artifactId>
-	<version>1.12.2</version>
+	<version>1.12.3</version>
 	<name>ProjectBrowser Portlet</name>
 	<url>http://github.com/qbicsoftware/projectbrowser-portlet</url>
 	<description>Browse and manage biomedical projects</description>
 	<packaging>war</packaging>
 	<!-- we only need to tell maven where to find our parent pom and other QBiC 
 		dependencies -->
 	<properties>
-		<vaadin.version>7.7.28</vaadin.version>
-		<vaadin.plugin.version>7.7.28</vaadin.plugin.version>
+		<vaadin.version>7.7.29</vaadin.version>
```

Comment:
why did you increase this version to .29? in your description you write you were able to build with .28?

Reply:
Should be `7.7.28` IMHO

Reply:
Can you confirm that you can build it in 7.7.28? then i'll downgrade it since i can't build it with 7.7.28